### ISSUE TITLE
new project

### ISSUE THREAD

**Issue by kenny089** [Jire/Overwatcheat#42]
Jire. Can you contact me. I will not take more than 3mins of your time.
kennethsnettbutikk@gmail.com
Thank you.

**Comment by Jire** [comment]
What's up?

**Comment by kenny089** [comment]
Hey bud. I wanted to reach out to you after this great Ow cheat. I used it from you publish it and i'am impressed by your work and got me inspired to learn Kotlin. Can you point me in a direction where i can go and learn what you just made? Or is it possible to somehow copy your source and rewrite it? Or will there be any issues due that Blizzard allready detected the source code itself? Am very new to this and i want to learn it step by step.

**Comment by Lonerw0lf** [comment]
Yes you can rewrite it like I did and made an universal pixel aim-bot out of modifying Overwatcheat and adding Charlatano's config features (along with adding a customizable trigger-bot), the "difficult" part for most people to understand should be the img to rgb (BufferedImageToRGB.kt), although you wouldn't need to change anything on these parts, the rest should be relatively easy for anyone to understand and or modify (with a bit of coding sense and basic Kotlin syntax).
And about Blizzard's Warden detecting the source code 🤔, not sure if they can detect code in a running jar file, you can google that, and if they can detect you can just randomize the strings and method functions and stuff like that.

**Comment by v-b7** [comment]
@Lonerw0lf Hi there, I'm working on my own version and I'm curious about some things.
Does your modified script include a smoothed/humanized aim? In current state it looks like it jumps straight to enemy's head. This will generate a high crit accuracy and blizz might flag you for review.
Also, for how long have you been using this. From what others say, blizzard can detect injected mouse events and screen-block/ban you because of this. Have you had any issues?
